tracer/src/Datadog.Trace/ServiceFabric/ServiceRemotingService.cs

```diff
@@ -105,5 +98,18 @@ private static void ServiceRemotingServiceEvents_SendResponse(object? sender, Ev
 
             ServiceRemotingHelpers.FinishSpan(e, SpanKinds.Server);
         }
+
+        private readonly struct ServiceRemotingRequestMessageHeaderSetter : ICarrierGetter<IServiceRemotingRequestMessageHeader>
```

Comment:
```suggestion
        private readonly struct ServiceRemotingRequestMessageHeaderGetter : ICarrierGetter<IServiceRemotingRequestMessageHeader>
```

Reply:
done in [bda286b](https://github.com/DataDog/dd-trace-dotnet/pull/2536/commits/bda286b6b2ed0deba92ad6aa8fb58fe700ba33e5)